<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>아이디 찾기</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>정보 입력</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>23</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>이       름 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>50</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>생년 월일 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBirthYear">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>47</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMajor">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>77</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>80</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>학      과 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>찾기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>310</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>홈으로</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfindID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
